server/embed/config.go

```diff
@@ -402,6 +402,9 @@ type Config struct {
 	// ExperimentalTxnModeWriteWithSharedBuffer enables write transaction to use a shared buffer in its readonly check operations.
 	ExperimentalTxnModeWriteWithSharedBuffer bool `json:"experimental-txn-mode-write-with-shared-buffer"`
 
+	// UnsafeAllowClusterVersionDowngrade enables only one single minor cluster version downgrade at a time automatically.
```

Comment:
```
// UnsafeAllowClusterVersionDowngrade is "true" to allow cluster version downgrade.
// "false" by default, since newer minor versions may introduce incompatible feature changes.
// For instance, lease checkpointer request to 3.4 will fail the remaining 3.3 nodes.
// But, if one does not use "lease checkpointer" feature, it can be safe to run 3.3 along with 3.4.
```